src/org/usfirst/frc/team2706/robot/commands/BatteryReader.java

```diff
@@ -0,0 +1,76 @@
+package org.usfirst.frc.team2706.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.PowerDistributionPanel;
+
+import org.usfirst.frc.team2706.robot.Robot;
+import org.usfirst.frc.team2706.robot.RobotMap;
+
+import edu.wpi.first.wpilibj.Timer;
+
+public class BatteryReader extends Command {
+
+
+    public static double batteryOutputVoltage;
+    // Used to tell us if the battery level is critical
```

Comment:
There should be a space right before the comment line so we know which line its referring to and it looks nicer